### ISSUE TITLE
PHP 7.4 compatibility

### ISSUE THREAD

**Issue by Qrzysio** [auraphp/Aura.Auth#102]
There is problem with AuraAuth in PHP 7.4. 

The algorithm is no longer integer. Now it's a string. 

So to fix the warning `hash(): Unknown hashing algorithm: 2y in (...)\aura\auth\src\Verifier\PasswordVerifier.php on line 57` I have changed this method:

```php
    public function verify($plaintext, $hashvalue, array $extra = array())
    {
        if (is_string($this->algo)) {
            return hash($this->algo, $plaintext) === $hashvalue;
        } else {
            return password_verify($plaintext, $hashvalue);
        }
    }
```

this way:

```php
    public function verify($plaintext, $hashvalue, array $extra = array())
    {
        return password_verify($plaintext, $hashvalue);
    }
```

Is any official update possible?

**Comment by ursoforte** [comment]
Great, but you can try in your code:
`$hash    = new PasswordVerifier(PASSWORD_DEFAULT);`
by it
`new PasswordVerifier(null);`

works too.

**Comment by brandonsavage** [comment]
There's a PR that fixes this, it's #103. Should be merged soon with 7.4 and 8.0/8.1 support.

**Comment by brandonsavage** [comment]
This is fixed in 4.0.3, released today. Please review and let us know if you have any further issues.
